tgui/packages/tgui/interfaces/Cargo.js

```diff
@@ -370,57 +369,93 @@ const CargoCartButtons = (props, context) => {
+          Quantity
+        </Stack.Item>
+        <Stack.Item ml="72px">
+          <CargoCartButtons />
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
 const CargoCart = (props, context) => {
   const { act, data } = useBackend(context);
   const { requestonly, away, docked, location, can_send } = data;
   const cart = data.cart || [];
   return (
-    <Section title="Current Cart" buttons={<CargoCartButtons />}>
+    <Section fill>
+      <CartHeader />
       {cart.length === 0 && <Box color="label">Nothing in cart</Box>}
       {cart.length > 0 && (
         <Table>
           {cart.map((entry) => (
             <Table.Row key={entry.id} className="candystripe">
-              <Table.Cell collapsing color="label">
-                #{entry.id}
+              <Table.Cell collapsing color="label" inline width="210px">
+                #{entry.id}&nbsp;{entry.object}
               </Table.Cell>
-              <Table.Cell>{entry.object}</Table.Cell>
-              <Table.Cell collapsing>
-                {!!entry.paid && <b>[Paid Privately]</b>}
+              <Table.Cell inline ml="65px" width="40px">
+                {(can_send && entry.can_be_cancelled && (
+                  <RestrictedInput
+                    width="40px"
+                    minValue={0}
+                    maxValue={50}
+                    value={entry.amount}
+                    onEnter={(e, value) =>
+                      act('modify', {
+                        order_name: entry.object,
+                        amount: value,
+                      })
+                    }
+                  />
+                )) || <Input width="40px" value={entry.amount} disabled />}
               </Table.Cell>
-              {(entry.dep_order && (
-                <Table.Cell collapsing textAlign="right">
-                  {formatMoney(entry.cost)} {entry.cost_type} earned on delivery
-                </Table.Cell>
-              )) || (
-                <>
-                  <Table.Cell collapsing textAlign="right">
-                    {formatMoney(entry.cost)} {entry.cost_type}
-                  </Table.Cell>
-                  <Table.Cell collapsing>
-                    {can_send && !!entry.can_be_cancelled && (
-                      <Button
-                        icon="minus"
-                        onClick={() =>
-                          act('remove', {
-                            id: entry.id,
-                          })
-                        }
-                      />
-                    )}
-                  </Table.Cell>
-                </>
-              )}
+              <Table.Cell inline ml="5px" width="10px">
+                {(can_send && entry.can_be_cancelled && (
+                  <Button
+                    icon="plus"
+                    onClick={() =>
+                      act('add_by_name', { order_name: entry.object })
+                    }
+                  />
+                )) ||
+                  ''}
```

Comment:
was this causing a formatting issue? shouldn't need to use the `|| ""` here

Reply:
Its actually needed cause it displays 0 on the UI if the condition is false

Reply:
ooo how did I miss that. cast the vars using `!!`  so you don't have to do this

Reply:
I'm not following. could you give an example. sorry, this is new to me

Reply:
Never mind i got it